llama_hub/llama_packs/arize_phoenix_query_engine/base.py

```diff
@@ -0,0 +1,91 @@
+"""
+Arize-Phoenix LlamaPack.
+"""
+
+from types import ModuleType
+from typing import TYPE_CHECKING, Any, Dict, List
+
+from llama_index import set_global_handler
+from llama_index.indices.vector_store import VectorStoreIndex
+from llama_index.llama_pack.base import BaseLlamaPack
+from llama_index.schema import TextNode
+
+if TYPE_CHECKING:
+    from phoenix import Session as PhoenixSession
+
+
+class ArizePhoenixQueryEnginePack(BaseLlamaPack):
+    """
+    The Arize-Phoenix LlamaPack show how to instrument your LlamaIndex query
+    engine for tracing and observability for your LLM application. It launches
```

Comment:
```suggestion
    The Arize-Phoenix LlamaPack show how to instrument your LlamaIndex query
    engine wth tracing. It launches
```

Reply:
addressed in [9911a6b](https://github.com/logan-markewich/llama-hub/pull/2/commits/9911a6ba9c8b1a261a07a2e3a86cd30aa00d259e)